_posts/2022-10-08-arrow-parquet-encoding-part-2.md

```diff
@@ -149,7 +149,7 @@ A definition level of `0` would imply a null at the level of `d`:
 }
 ```
 
-A definition level of `1` would imply a null at the level of `d`
+A definition level of `1` would imply a null at the level of `d.d2`
```

Comment:
@alamb @tustvold   i think this should be `d.d2`

Reply:
Yes I think you are correct

Reply:
Yes we should probably also change the example to be valid JSON

```
"d": { }
```